pkg/vm/engine/types.proto

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2021 Matrix Origin
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+package engine;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
```

Comment:
this is completely wrong.    We cannot depend on gogoproto in anyway.

Reply:
But our project takes gogoproto as the default plugin of PB, see [proto/gen.sh](https://github.com/matrixorigin/matrixone/blob/352fa7db1643661ea83671af2088fda57a26418d/proto/gen.sh#L108).